src/pyOpenMS/pxds/BayesianProteinInferenceAlgorithm.pxd

```diff
@@ -77,13 +76,12 @@ cdef extern from "<OpenMS/ANALYSIS/ID/BayesianProteinInferenceAlgorithm.h>" name
                                          bool greedy_group_resolution,
                                          ExperimentalDesign exp_des) nogil except +
           # wrap-doc:
-          #   Performs inference
-          #   -----
           #   Writes its results into protein and (optionally also) peptide hits (as new score).
```

Comment:
Too many whitespaces after the "#". Two are substracted for indentation with the wrap- statements. So you are left with another one.

Reply:
Probably in many other places as well.